docs/data/base/components/button/UnstyledButtonIntroduction/system/index.js

```diff
@@ -58,23 +58,21 @@ const Button = styled(BaseButton)(
   &:active {
     background-color: ${blue[700]};
     box-shadow: none;
+    transform: scale(0.99);
```

Comment:
It adds depth to the click interaction 🤤

Reply:
cc @zanivan to experiment/polish/standardize!